graphql/image_test.go

```diff
@@ -65,6 +65,21 @@ func TestToolchains(t *testing.T) {
 	assert.Equal(t, testToolchain, utility.FromStringPtr(res[0].Name))
 }
 
+func TestEvents(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestEvents")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+	imageID := "amazon2"
+	image := model.APIImage{
+		ID: &imageID,
+	}
+	res, err := config.Resolvers.Image().Events(ctx, &image, 10, 0)
+	require.NoError(t, err)
+	assert.NotEmpty(t, res)
```

Comment:
could we also verify the len?

Reply:
it should verify the limit works in a variety of cases like no events, high limit, high limit and a lot of events, small limit a lot of events, etc

Reply:
Added a test to verify that it errors for an empty image, a test to check that the limit is satisfied, and a test to ensure that AMI values are different on different pages. In the database, there do not seem to be many images with a lot of events (max 15ish), so I think testing with a high limit might not be helpful.